keras_nlp/layers/token_and_position_embedding.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Creates an Embedding Layer and adds Positional Embeddings"""
+
+from tensorflow import keras
+
+import keras_nlp.layers
+
+
+class TokenAndPositionEmbedding(keras.layers.Layer):
+    """A layer which sums a token and position embedding.
+
+    This class assumes that in the input tensor, the last dimension corresponds
```

Comment:
I think this description is incorrect. In general the assumption should be an input of shape `(batch_dim, sequence_dim)` right? Assumption--the last dimension is the sequence dim.

And the output will be `(batch_dim, sequence_dim, embedding_dim)`.

Reply:
Yes, after I updated the tests, this seems to be the incorrect. This will be fixed.